### ISSUE TITLE
Add a Rake Task for Setting up IT Dependencies

### ISSUE THREAD

**Issue by original-brownbear** [elastic/logstash#7135]
There should be a Rake task for setting up (and especially for refreshing) the dependencies of the tests under `qa/integration`. Going through 

```md
## Preparing a test run

1. If you already have a LS binary in `LS_HOME/build/logstash-<version>`, skip to step 5
2. From Logstash git source directory or `LS_HOME` run `rake artifact:tar` to build a package
2. Untar the newly built package
3. `cd build`
4. `tar xvf logstash-<version>.tar.gz`
5. `cd LS_HOME/qa/integration`
6. `bundle install`: This will install test dependency gems.
```

by hand is really bothersome whenever you want to run against updated LS core code. This should just be a one off rake target.

**Comment by ph** [comment]
Actually we have something for that its 2 scripts in the `ci/` directory

https://github.com/elastic/logstash/blob/master/ci/travis_integration_install.sh
https://github.com/elastic/logstash/blob/master/ci/travis_integration_run.sh

You can see the task that are run on ci there https://github.com/elastic/logstash/blob/master/.travis.yml#L13-L31

**Comment by original-brownbear** [comment]
@ph yea, I guess it would be nice to make those a little portable and one off without the need for the env variable + document them here right ?

**Comment by ph** [comment]
I believe we could do the following, remove the environment variable from the shell scripts and only use it in the `.travis` to be used in the test matrix definition and merge the shell script into 1?

**Comment by original-brownbear** [comment]
@ph sounds good :)

**Comment by original-brownbear** [comment]
implemented via #8599
